### DISCUSSION TITLE
Blank screen after shutdown

### DISCUSSION THREAD

**Discussion by madmaximux** [chad-ermacora/rp-network-testers#14] [Ideas]
I decided to make a new discussion to represent a new idea - will prob make it easier in commits to reference etc.

Could we have the screen go blank after shutting down? Like can it display that shutdown message for a bit, but then go away before final commands are sent? Or maybe the message is changed to ... "please shutdown after 15 seconds.. this message will disappear now..". Just thinking out loud again.

I haven't used the tester in a while and it's been on [this](https://i.imgur.com/qkZJq20.jpeg) screen since. I know it's e-ink and doesn't use any energy, but I've comments where others suggest complete clearing up of the screen before turning off (for longevity ?).

**Comment by chad-ermacora** [comment]
I'm not against having a nice blank screen after shutdown, but I'm worried people will see the blank screen and then pull the power right away (possibly corrupting the SD card), especially if they get distracted and miss the original shutdown message. It would also add extra time to show the message before the system even starts to shut down. 

E-Ink devices don't suffer from burn-in effect, only temporary ghosting, so I don't think longevity is being affected. I'm pretty sure the wear and tear only happens during the refresh process. 

I used to have a timestamp display, so you could see when 15 seconds has passed, but since the Pi's don't have a real-time clock (It doesn't keep time after shutdown), the date is often horribly wrong when you are not connected to a network with internet access on boot.
